indexer/packages/postgres/src/stores/affiliate-info-table.ts

```diff
@@ -92,3 +93,32 @@ export async function findById(
     .findById(address)
     .returning('*');
 }
+
+export async function paginatedFindWithAddressFilter(
```

Comment:
can you add a test that checks if no `AffiliateInfoFromDatabase` matches the query that it returns undefined? Ideally it would return an empty list, that seems like it would make more sense to me

Reply:
Most find methods in tables return undefined but I agree that empty list makes more sense. I can check for undefined and convert to empty list before return. 

Will add test.